daemon/cluster/executor/container/controller.go

```diff
@@ -47,9 +47,15 @@ func newController(b executorpkg.Backend, task *api.Task, dependencies exec.Depe
 		return nil, err
 	}
 
+	// see https://github.com/moby/moby/pull/33377 for more information
```

Comment:
Its not good to reference issues. You should explain what the code does in line.